File: app/src/main/java/com/example/samson/immvrse/news_feed/NewsFeedAdapter.java
package com.example.samson.immvrse.news_feed;

import android.content.Context;
import android.content.Intent;
import android.databinding.DataBindingUtil;
import android.support.annotation.NonNull;
import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.example.samson.immvrse.R;
import com.example.samson.immvrse.VideoViewerActivity;
import com.example.samson.immvrse.databinding.ViewNewsFeedBinding;


public class NewsFeedAdapter extends RecyclerView.Adapter<NewsFeedViewHolder> {

    private Context context;

    public NewsFeedAdapter(Context context) {
        this.context = context;
    }

    @NonNull
    @Override
    public NewsFeedViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType) {
        ViewNewsFeedBinding binding = DataBindingUtil.inflate(LayoutInflater.from(parent.getContext()),R.layout.view_news_feed, parent, false);
        return new NewsFeedViewHolder(binding.getRoot());
    }

    @Override
    public void onBindViewHolder(@NonNull NewsFeedViewHolder holder, int position) {
        holder.getBinding().newsFeedImage.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(context, VideoViewerActivity.class);
                context.startActivity(intent);
            }
        });
        }

    @Override
    public int getItemCount() {
        return 1;
    }
}

File: app/src/main/java/com/example/samson/immvrse/news_feed/NewsFeedAdapter.kt
package com.example.samson.immvrse.news_feed

import android.content.Context
import android.content.Intent
import android.support.v7.widget.RecyclerView
import android.view.LayoutInflater
import android.view.ViewGroup
import com.example.samson.immvrse.VideoViewerActivity
import kotlinx.android.synthetic.main.view_news_feed.view.*

class NewsFeedAdapter(private val context: Context) : RecyclerView.Adapter<NewsFeedViewHolder>() {

    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): NewsFeedViewHolder {
        return NewsFeedViewHolder(LayoutInflater.from(parent.context).inflate(viewType, parent, false))
    }

    override fun onBindViewHolder(holder: NewsFeedViewHolder, position: Int) {
        holder.itemView.news_feed_image.setOnClickListener {
            val intent = Intent(context, VideoViewerActivity::class.java)
            context.startActivity(intent)
        }
    }

    override fun getItemCount(): Int {
        return 1
    }
}
